charter.md

```diff
@@ -0,0 +1,74 @@
+  - Defining and staffing release roles to manage the resolution of release blocking criteria
+  - Defining and driving development processes (e.g. merge queues, cherrypicks) and release processes
+    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
+  - Managing the creation of release specific artifacts, including:
+    - Code branches
+    - Binary artifacts
+    - Release notes
+- Continually improving release and development processes
+  - Working closely with SIG Contributor Experience to define and build tools to facilitate release process (e.g. dashboards)
+  - Working closely with SIG Testing to determine and implement tests, automation, and labeling required for stable releases
+  - Working with downstream communities responsible for packaging Kubernetes releases
+  - Working with other SIGs to agree upon the responsibilities of their SIG with respect to the release
+  - Defining and collecting metrics related to the release in order to measure progress over each release
+  - Facilitating release retrospectives in conjunction with SIG PM
+- Collaborating with downstream communities which build artifacts from Kubernetes releases
+  - Ensuring the appropriate level of integration with publishing the build artifacts
+
+### Out of scope
+
+#### Support
+
+SIG Release itself is not responsible for end user support or creation of patches for support streams. There are support forums for end users to ask questions and report bugs, subject matter experts in other SIGs triage and address issues and when necessary mark bug fixes for inclusion in a patch release.
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance] and opts-in to updates and modifications to [sig-governance].
+
+Specifically, the common guidelines (see: [sig-governance]) for continuity of membership within roles in the SIG are followed.
+
+### Deviations from [sig-governance]
+
+- SIG Release does not have top-level SIG Technical Leads; SIG Release Chairs can serve in both an operational and technical capacity
+- SIG Release subprojects have subproject chairs
+
+#### SIG Membership
+
+Additionally, SIG Release has a concept of membership. SIG members can be occasionally called on to assist with decision making, especially as it relates to gathering historical context around existing policies.
+
+While a strict definition of membership has yet to be established, we will initially seed a set of members from the list of all documented former Release Team members and subproject OWNERS.
+
+SIG Release Members should be the first group of community members considered when seeking to fill additional roles across SIG Release proper and SIG Release subprojects, except where explicitly defined in other policy.
+
+### Subproject Creation
+
+- Subprojects must be created by [KEP] proposal and accepted by [lazy-consensus] with fallback on majority vote of
```

Comment:
I believe it that subproject creation, along with most other major decisions, should follow the same process:

1. seek lazy consensus around a KEP
1. in the event consensus cannot be reached -> majority decision by SIG Chairs
1. SIG members _may_ override the majority decision of SIG Chairs by a supermajority (60%)

this may sound like process for process sake but I really feel that it's important to take the position

*  the people who do the work make the decisions (membership is derived from `OWNERS` and the ultimate authority is held my SIG members)
* SIG Chairs are supposed to represent the sentiment of and facilitate the decision making by SIG Members

Reply:
@calebamiles -- Added that language.

Reply:
@tpepper -- the way I imagine the RT is a single subproject with rotating membership and the current RT Lead + Shadow acting as subproject chairs. SIG Release Chairs would be considered subproject chairs here for the sake of continuity across release cycles.

Let's bounce ideas around for that though when we draft the KEP for RT.